frontend/src/wizards/RoleAssignment/ProjectsTable.tsx

```diff
@@ -45,73 +44,49 @@ export function ProjectsTable({
   )
 
   const projectsData: ProjectTableData[] = useMemo(() => {
```

Comment:
didn't test this refactoring, I'm pretty confident about it, I would TDD it and check whether my proposal remains green